dask/tests/test_base.py

```diff
@@ -8,6 +8,8 @@
 
 from tlz import merge, partial, compose, curry
 
+from unittest import mock
```

Comment:
Not entirely sure what convention to employ here in terms of whether to `import mock` at this level or within `test_normalize_object` itself (just as it's been done for `import warnings`; the latter has been imported within the test only, because its general utility is not as wide, in this particular context).

Any feedback would be greatly appreciated.